worlds/saving_princess/docs/en_Saving Princess.md

```diff
@@ -0,0 +1,56 @@
+# Saving Princess
+
+## Quick Links
+- [Setup Guide](/tutorial/Saving%20Princess/setup/en)
+- [Options Page](/games/Saving%20Princess/player-options)
+- [Saving Princess Archipelago GitHub](https://github.com/LeonarthCG/saving-princess-archipelago)
+
+## What changes have been made?
+
+The game has had several changes made to add new features and prevent issues. The most important changes are the following:
+- There is an in-game connection settings menu, autotracker and client console.
+- New save files are created and used automatically for each seed and slot played.
+- The game window can now be dragged and a new integer scaling option has been added.
+
+## What items and locations get shuffled?
+
+The chest contents and special weapons are the items and locations that get shuffled.
+
+Additionally, there are new items to work as filler and traps, ranging from a full health and ammo restore to spawning a Ninja on top of you.
+
+The Expanded Pool option, which is enabled by default, adds a few more items and locations:
+- Completing the intro sequence, powering the generator with the Volt Laser and defeating each boss become locations.
+- 4 Keys will be shuffled, which serve to open the door to the final area in place of defeating the main area bosses.
+- A System Power item will be shuffled, which restores power to the final area instead of this happening when the generator is powered.
+
+## What does another world's item look like in Saving Princess?
+
+Some locations, such as boss kills, have no visual representation, but those that do will have the Archipelago icon.
+
+Once the item is picked up a textbox will inform you of the item that was found as well as the player that will be receiving it.
+
+These textboxes will have colored backgrounds and comments about the item category.
+For example, progression items will have a purple background and say "Looks plenty important!".
+
+## When the player receives an item, what happens?
+
+When you receive an item, a textbox will show up.
+This textbox shows both which item you got and which player sent it to you.
+
+If you send an item to yourself, however, the sending player will be omitted.
+If the items are cheated in, no textbox will be displayed.
```

Comment:
By cheated in I meant anything that has the server as the player that found it. So, yeah, start_inventory would count for that. I found it annoying to see all the pop-ups when using start_inventory, but perhaps this would just be confusing instead?

I don't feel strongly one way or the other, so if this is outside of the norm of what other worlds do then I'm happy to change it.